client/src/views/admin/addCountry.vue

```diff
@@ -0,0 +1,189 @@
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <b-modal class="modal-header-waring" id="modal-center" v-model="modalShowView" centered hide-footer>
+            <div>
+                <div class="form mb-3">
+                    <b-form-input v-model="updateCountry.name" placeholder="Enter the CountryName"></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="updateCountry.description" placeholder="Description"></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="updateCountry.countryCode" placeholder="Country Code"></b-form-input>
+                </div>
+                <b-button variant="outline-primary" size="lg" @click="handleSave()">Save</b-button>
+            </div>
+        </b-modal>
+        <b-modal class="modal-header-waring" id="modal-center" v-model="deleteModalShowView" centered hide-footer>
+            <div class="d-block text-center">
+                <h3>Are you sure want to Delete</h3>
+            </div>
+            <div class="form mb-3 text-center">
+                <b-button type="button" class="btn btn-danger" @click="deleted">OK</b-button>
+            </div>
+
+        </b-modal>
+    </div>
+</template>
+<script>
+export default {
+    name: "addCountry",
+    data() {
+        return {
+            modalShowView: false,
+            deleteModalShowView: false,
+            deleteId: "",
+            updateCountry: "",
+            selectedId: "",
+            country: { name: "", description: "", countryCode: "" },
+            addedCountries: [],
+            countryValues: {},
+            root: process.env.VUE_APP_ROOT_API,
+        };
+    },
+    created() {
+        this.test()
+    },
+    methods: {
+        test() {
+            this.$axios
+                .get(`${this.root}/country`)
+                .then((response) => {
+                    if (response.data.status) {
+                        this.addedCountries = response.data.data
+                    }
+                })
+                .catch((err) => {
+                    console.log(err);
+                    this.$bvToast.toast("Error Occured!", {
+                        title: "Error",
+                        variant: "danger",
+                        solid: true,
+                    });
+                })
+        },
+        handleUpdate(data) {
+            this.selectedId = data.id;
+            this.updateCountry = {
+                name: data.name,
+                description: data.description,
+                countryCode: data.countryCode,
+            };
+            this.modalShowView = true;
+        },
+        handleSave() {
+            this.modalShowView = false;
+
+            this.$axios
+
+                .patch(`${this.root}/country/` + this.selectedId, this.updateCountry)
+                .then((response) => {
```

Comment:
please refer how to `patch` data for country `/server/test-rest-api/countrytest.rest` file